src/o365/spo/commands/web/web-set.ts

```diff
@@ -25,9 +25,13 @@ interface Options extends GlobalOptions {
   title?: string;
   webUrl: string;
   footerEnabled?: string;
+  searchScope?: string;
 }
 
 class SpoWebSetCommand extends SpoCommand {
+  private static searchScopeOptions: string[] =          
+    ['DefaultScope', 'Tenant', 'Hub', 'Site'];
```

Comment:
I would recommend making the options case insensitive. I would turn the values in the array into lowercase and then compare the options.toLowerCase() against the array when a check is needed.

Reply:
Hi @VelinGeorgiev . This has been added in my checkin.  Thanks.